applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -411,6 +411,24 @@ static void le_audio_evt_handler(enum le_audio_evt_type event)
 		LOG_DBG("Bitrate: %d kbps", bitrate);
 		break;
 
+	case LE_AUDIO_EVT_PRES_DELAY_SET:
+		uint32_t pres_delay;
+
+		ret = le_audio_config_get(NULL, NULL, &pres_delay);
```

Comment:
Should these prints be errors?

Reply:
Changed to ERR in streamctrl, but not underlying modules